# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/frotz/no_rootcheck.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- frotz-2.54/src/curses/ux_init.c.vanilla	2023-03-09 23:10:20.221000000 +0000
+++ frotz-2.54/src/curses/ux_init.c	2023-03-09 23:10:42.978000000 +0000
@@ -204,12 +204,6 @@
 
 	zoptarg = NULL;
 
-#if !defined(WIN32) && !defined(__HAIKU__)
-	if ((getuid() == 0) || (geteuid() == 0)) {
-		printf("I won't run as root!\n");
-		os_quit(EXIT_FAILURE);
-	}
-#endif
 
 #ifdef WIN32
 #define HOMEDIR "USERPROFILE"
